resources/workflow_options/fresh.run.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Oh, super cool - I hadn't thought of having a JSON specifically for bypassing call-caching and starting a new run.  Nice!